limited/sly fox

```diff
@@ -0,0 +1,10 @@
+**Sly Fox** | Werewolf Power | Limited
+__Basics__
+Each day the Sly Fox may choose a player and hide in their house, protecting them from any attack targeted at them but not the house they are hidden in, the wolfpack is informed at the start of the night where the Sly Fox is hidden. Once per game, the Sly Fox may alternatively join the wolfpack as a regular wolf. 
```

Comment:
I think the phrasing of all of this should probably be closer to how Hooker is phrased. There also a bunch of places where the grammar is odd right now.